src/extension/wps/src/main/java/au/org/emii/wps/gogoduck/index/FeatureSourceIndexReader.java

```diff
@@ -68,7 +73,10 @@ public FeatureSourceIndexReader(Catalog catalog, Map<String, String> urlSubstitu
             while (iterator.hasNext()) {
                 SimpleFeature feature = iterator.next();
                 String url = (String) feature.getAttribute(urlField);
-                long size = Math.round((Double)feature.getAttribute(sizeField));
+                long size = 0;
```

Comment:
Not sure why this was required - unless it was to use the test data.  We should change the test data to be more realistic if that's the case.